batch/Makefile

```diff
@@ -69,13 +69,18 @@ test-local: check
 endif
 test-local: install-cloud-sql-proxy 
 test-local: batch-secrets/batch-test-cloud-sql-config.json
-	POD_NAMESPACE='test' BATCH_USE_KUBE_CONFIG=1 CLOUD_SQL_PROXY=1 ./test-locally.sh
+	POD_NAMESPACE='test' BATCH_USE_KUBE_CONFIG=1 ./test-locally.sh
```

Comment:
I think this needs CLOUD_SQL_CONFIG_PATH. There are a few other rules where ./test-locally.sh is called, but doesn't have CLOUD_SQL_CONFIG_PATH set

Reply:
It's defined in `test-locally.sh` and depends on `$IN_HAIL_CI`

```
if [[ -z $IN_HAIL_CI ]]; then
    export CLOUD_SQL_CONFIG_PATH=`pwd`/batch-secrets/batch-test-cloud-sql-config.json
    connection_name=$(jq -r '.connection_name' $CLOUD_SQL_CONFIG_PATH)
    host=$(jq -r '.host' $CLOUD_SQL_CONFIG_PATH)
    port=$(jq -r '.port' $CLOUD_SQL_CONFIG_PATH)
    ./cloud_sql_proxy -instances=$connection_name=tcp:$port &
    proxy_pid=$!
    ../until-with-fuel 30 curl -fL $host:$port
else
    export CLOUD_SQL_CONFIG_PATH=/batch-secrets/batch-test-cloud-sql-config.json
fi
```

Reply:
Right, but your test depends on this file, which may or may not be present.

Reply:
It is there though as a dependency `test-local: batch-secrets/batch-test-cloud-sql-config.json`. I don't see what the issue is.

Reply:
Ah, you're right. I think just below in `test-local-in-cluster` then